src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,365 @@
+In this scenario you want to contain all kubernetes records under a subdomain
+of a domain you host in Route53.  This requires creating a second hosted zone
+in route53, and then setting up route delegation to the new zone.
 
+In this example you own `example.com` and your records for Kubernetes would
+look like `etcd-us-east-1c.internal.clustername.kubernetes.example.com`
+
+This is copying the NS servers of your **SUBDOMAIN** up to the **PARENT**
+domain in Route53.  To do this you should:
+
+
+```bash
+ID=$(uuidgen) && aws route53 create-hosted-zone --name subdomain.example.com --caller-reference $ID | jq .DelegationSet.NameServers
 ```
-export KUBE_AWS_ZONE=us-west-1c
-export KUBERNETES_PROVIDER=aws
+
+* Note your **PARENT** hosted zone id
+
+```bash
+# Note: This example assumes you have jq installed locally.
+aws route53 list-hosted-zones | jq '.HostedZones[] | select(.Name=="example.com.") | .Id'
 ```
 
-For evaluation, we find that the t2 instance classes are a reasonable bang for the buck. Do note that the t2 class does
-track CPU credits. Performance of your evaluation cluster may be impacted when you exhaust the CPU credit limit. Select
-your instance sizes and worker count.
+* Create a new JSON file with your values (`subdomain.json`)
+
+Note: The NS values here are for the **SUBDOMAIN**
 
 ```
-export MASTER_SIZE=t2.medium
-export NODE_SIZE=t2.large
-export NUM_NODES=2
-export NODE_ROOT_DISK_SIZE=100
+{
+  "Comment": "Create a subdomain NS record in the parent domain",
+  "Changes": [
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "subdomain.example.com",
+        "Type": "NS",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ns-1.awsdns-1.co.uk"
+          },
+          {
+            "Value": "ns-2.awsdns-2.org"
+          },
+          {
+            "Value": "ns-3.awsdns-3.com"
+          },
+          {
+            "Value": "ns-4.awsdns-4.net"
+          }
+        ]
+      }
+    }
+  ]
+}
 ```
 
-Last, so you can easily identify instances in the AWS Console, specify an instance prefix:
+* Apply the **SUBDOMAIN** NS records to the **PARENT** hosted zone.
+
 ```
-export KUBE_AWS_INSTANCE_PREFIX=first-k8s
+aws route53 change-resource-record-sets \
+ --hosted-zone-id <parent-zone-id> \
+ --change-batch file://subdomain.json
 ```
 
-## Set up kubectl CLI
+Now traffic to `*.example.com` will be routed to the correct subdomain hosted zone in Route53.
 
-We will need to use `kubectl` to check everything is running smoothly, so let's get it on the $PATH.
+You can now skip to [testing your DNS setup](#testing-your-dns-setup)
 
+#### Scenario 2: Setting up Route53 for a domain purchased with another registrar
+
+If you bought your domain elsewhere, and would like to dedicate the entire domain to AWS you should follow the guide [here](http://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-transfer-to-route-53.html)
```

Comment:
That is okay in my mind. We make several external links to https://devcenter.heroku.com/ and to docs.aws.amazon.com elsewhere as well, so it wouldn't be the first time.